src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml

```diff
@@ -281,7 +281,7 @@
     <sys:String x:Key="Activate_Product_Key">Produktschlüssel</sys:String>
     <sys:String x:Key="Activate_Product_Key.Description">Office-Produktschlüssel installieren, Produktschlüsselformat: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</sys:String>
     <sys:String x:Key="Activate_Install_Product_Key">Schlüssel _installieren</sys:String>
-    <sys:String x:Key="Activate_Uninstall_Unactivated_Key">_Uninstall all unactivated keys</sys:String>
+    <sys:String x:Key="Activate_Uninstall_Unactivated_Key">_Alle nicht aktivierten Schlüssel deinstallieren</sys:String>
```

Comment:
The access key `A` is same as the next string, please change to other one if possible.